ral-facilities/datagateway
# Ensure that datagateway tests will continue to work with new datagateway-api config

## Description
Datagateway-api will begin using a new configuration file soon. This pr ensures that the new configuration would be supported.
## Testing instructions
Ensure that you have checked out datagateway-api branch feature/use-yaml-config#372

- [ ] Merge https://github.com/ral-facilities/datagateway-api/pull/388
- [ ] Change branch to checkout datagateway-api main.

## Agile board tracking
connect to #372
